.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "third_party/protobuf"]
+	path = third_party/protobuf
+	url = https://github.com/google/protobuf.git
+	ignore = dirty
```

Comment:
Are you fixing the version/tag that is checked out of the submodule anywhere? The previous shell script always checked out version 3.6.1.

Reply:
Yes, if you scroll to the very bottom, you will see `Submodule protobuf added at 48cb18`... `48cb18` corresponds to `v3.6.1`